### PR TITLE
Add v0 of the README

### PR DESCRIPTION
**Pull request by mertcelebi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Arcadia Science Software Handbook
+
+The repository is home to the core software engineering guides and policies at [Arcadia Science](https://www.arcadiascience.com/). If you're curious about where to begin, [the source control policy document](/guides-and-policies/policy-source-control) is a good place to start!
```
Comment (taylorreiter) [inline README.md]:
```suggestion
This repository is home to the core software engineering guides and policies at [Arcadia Science](https://www.arcadiascience.com/). If you're curious about where to begin, [the source control policy document](/guides-and-policies/policy-source-control) is a good place to start!
```

File: README.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+## Naming convention
```
Comment (taylorreiter) [inline README.md]:
```suggestion
## Naming conventions
```

File: README.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+- A **guide** document describes software engineering best practices for reusable and reproducible work. These are strong suggestions but exceptions to the rule can happen.
```
Comment (taylorreiter) [inline README.md]: strong suggestions vs. sensible defaults? Curious how strongly we want to push toward things written in guides.

**Review by taylorreiter — DISMISSED** [review]
Few suggestions/questions, but otherwise lgtm!

File: README.md (hunk 3 above)
Comment (mertcelebi) [inline README.md]:
```suggestion
- A **guide** document describes software engineering best practices for reusable and reproducible work. These are sensible defaults and exceptions to the rule are expected to happen.
```

Comment (mertcelebi) [reply README.md]: I like sensible defaults!

**Pull request merged**